features/google/src/components/awakeningLocationMinistory/awakeningLocationMinistory.jsx

```diff
@@ -0,0 +1,135 @@
+import "./awakeningLocationMinistory.css";
+import React, { useState } from "react";
+import SourceInfoButton from "../sourceInfoButton/sourceInfoButton.jsx";
+import i18n from "!silly-i18n";
+
+export const AwakeningLocationSummary = ({ dateData }) => {
+    const defaultDate = Object.entries(dateData).find(
+        (entry) => !!entry[1].firstActivity
+    )[0];
+    const [selectedDate, setSelectedDate] = useState(defaultDate);
+    const data = dateData[selectedDate];
+    const dateSelection = false;
+    const handleChangeDate = ({ target }) => {
+        const formattedDate = target.value
+            .split("-")
+            .reverse()
+            .map((e) => parseInt(e))
+            .join("-");
+        setSelectedDate(formattedDate);
+        console.log(formattedDate);
+    };
+    //could be used to make the text more human later on
+    // const week = [
+    //     "Sunday",
+    //     "Monday",
+    //     "Tuesday",
+    //     "Wednesday",
+    //     "Thursday",
+    //     "Friday",
+    //     "Saturday",
+    // ];
+    return (
+        <div className="awakening-location-ministory-summary ">
+            {dateSelection ? (
+                <div className="date-browser">
+                    <label>
+                        Enter a date:
```

Comment:
ObReference to translation strings

Reply:
as this is disabled right now, I will leave this for the next cycle when the design of the date picker will be decided by the designers